index.js

```diff
@@ -7,3 +7,23 @@ var index = _.groupBy(data, 'codePostal');
 exports.find = function(postalCode) {
     return index[postalCode] || [];
 };
+
+
+/** Communes with 0 inhabitants (“communes mortes pour la France”).
+ * A sensible dataset to use for testing in any application that has a precondition where postal codes should map to a physical person.
+ */
+var _deadCommunes;  // lazy load & memoize
+
+Object.defineProperty(exports, 'dead', {
+    get: function getDeadCommunes() {
+        if (! _deadCommunes) {
+            var DEAD_INSEE_CODES = [ '50173', '55039', '55050', '55239', '55307', '55139' ];
```

Comment:
[Équeurdreville-Hainneville (50173)](http://www.insee.fr/fr/themes/comparateur.asp?codgeo=com-50173) population is 17319.

But you missed [Fleury-devant-Douaumont (55189)](http://www.insee.fr/fr/themes/comparateur.asp?codgeo=com-55189) which is one in the sadest place of the world.